core/server/src/test/java/alluxio/master/file/PermissionCheckTest.java

```diff
@@ -465,7 +465,7 @@ public void readDirFailTest() throws Exception {
   public void setStateSuccessTest() throws Exception {
     // set unmask
     Configuration conf = MasterContext.getConf();
-    conf.set(Constants.SECURITY_AUTHORIZATION_PERMISSIONS_UMASK, "044");
+    conf.set(Constants.SECURITY_AUTHORIZATION_PERMISSIONS_UMASK, "000");
```

Comment:
Was this a bug before or its because we need more permissions for getFileInfo now?